components/layout_2020/display_list/stacking_context.rs

```diff
@@ -1366,17 +1373,68 @@ impl BoxFragment {
         Some(display_list.define_clip_chain(*parent_clip_chain_id, [clip_id]))
     }
 
-    fn build_scroll_frame_if_necessary(
+    fn used_overflow(&self) -> PhysicalVec<ComputedOverflow> {
+        let overflow = self.style.effective_overflow();
+        let is_replaced_emelent = self.base.flags.contains(FragmentFlags::IS_REPLACED);
+        let overflow_x = if is_replaced_emelent && overflow.x != ComputedOverflow::Visible {
+            ComputedOverflow::Clip
+        } else {
+            overflow.x
+        };
+
+        let overflow_y = if is_replaced_emelent && overflow.y != ComputedOverflow::Visible {
+            ComputedOverflow::Clip
+        } else {
+            overflow.y
+        };
+
+        PhysicalVec::new(overflow_x, overflow_y)
```

Comment:
Seems simpler like this:

```suggestion
        let mut overflow = self.style.effective_overflow();
        if self.base.flags.contains(FragmentFlags::IS_REPLACED) {
            if overflow.x != ComputedOverflow::Visible {
                overflow.x = ComputedOverflow::Clip;
            }
            if overflow.y != ComputedOverflow::Visible {
                overflow.y = ComputedOverflow::Clip;
            }
        }
        overflow
```

Reply:
Done